### ISSUE TITLE
Enable exporting of complex geometry to GDML

### ISSUE THREAD

**Issue by jintonic** [jintonic/shine#14]
Geant4 text geometry cannot handle [Tessellated Solids](https://geant4-userdoc.web.cern.ch/UsersGuides/ForApplicationDeveloper/html/Detector/Geometry/geomSolids.html#tessellated-solids). GDML can.

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/WO0Hp2xBdpTrQTcRpNhnhA37oNBgFo
Should I export these .tg file to .gdml?

**Comment by jintonic** [comment]
No. It is not our first priority to enable importing of tg files.

**Comment by jintonic** [comment]
Is this done?

**Comment by bestdeveloper23** [comment]
I've researched a lot about Tssellated Solid but I'm not clear.
Can you show me your app code for this?

**Comment by jintonic** [comment]
C++ side or GDML side?

**Comment by jintonic** [comment]
We don't create Tessellated solids in three.js. Why do we need to export such a thing to GDML from three.js?

**Comment by jintonic** [comment]
It might be useful if we want to IMPORT a GDML file that contains Tessellated solids. But in that case, we need to understand how Tessellated solids are defined in GDML, not in C++.

**Comment by bestdeveloper23** [comment]
That's clear.
So I need to that gdml file.

**Comment by jintonic** [comment]
I'll see if can I find a GDML file that contains Tessellated solid. Meanwhile, if you have no other things to work on, please work on the import/export of solids created by boolean operations.

**Comment by bestdeveloper23** [comment]
Okay.
